lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -316,8 +316,8 @@ def digital_wallet_payment_method?(payment_method)
         payment_method.source == :google_pay || payment_method.source == :apple_pay
       end
 
-      def adding_network_token_card_data?(payment_method)
-        return true if payment_method.is_a?(ActiveMerchant::Billing::NetworkTokenizationCreditCard) && payment_method.source == :network_token
+      def adding_network_token_card_data?(payment_method, options = {})
```

Comment:
I would prefer this stay as it was originally was. `adding_network_token_card_data?` is used in 4 other places and I do not want to alter how that is working at the moment.

Reply:
Sure, no problem :)